.gitignore

```diff
@@ -1,3 +1,4 @@
+.vscode
```

Comment:
Tool-specific configuration should be in the [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) on your device, not in every code base you work on.